WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderPostUiStateBuilder.kt

```diff
@@ -122,6 +125,10 @@ class ReaderPostUiStateBuilder @Inject constructor(
             post.takeIf { post.cardType == VIDEO }
                     ?.let { post.featuredVideo }
 
+    private fun buildExpandedTagsViewVisibility(post: ReaderPost) = post.isDiscoverPost
```

Comment:
I think I'll have to add a check for RI FF here so that tags do not display on develop. Will verify and add.

Reply:
Yep, I think you are correct.

Btw the new discover tab doesn't work with "real discover posts" -> "isDiscoverPost" will return false for them. "Real Discover post" is a post from "discover.wordpress.com" and the new feed will contain posts from various blogs.

Reply:
Updated here: 25fa018